bayesian/default_config.py

```diff
@@ -53,3 +53,5 @@
 GEMINI_SERVER_URL = "http://{host}:{port}".format(
                      host=os.getenv('F8A_GEMINI_SERVER_SERVICE_HOST'),
                      port=os.getenv('F8A_GEMINI_SERVER_SERVICE_PORT'))
+
+COMPONENT_ANALYSES_LIMIT = int(os.getenv('BAYESIAN_COMPONENT_TAGGED_COUNT', 10))
```

Comment:
The variable should be `COMPONENT_ANALYSES_LIMIT`

Reply:
oops 😬

Reply:
changed it. Thanks :)